rtree.go

```diff
@@ -486,6 +501,16 @@ func (tree *Rtree) NearestNeighbor(p Point) Spatial {
 	return obj
 }
 
+// GetAllBoundingBox returning slice of bounding boxes by traversing tree. Slice
+// includes bounding boxes from all non-leaf nodes.
+func (tree *Rtree) GetAllBoundingBox() []*Rect {
```

Comment:
could you rename to GetAllBoundingBoxes (and the helper function to getAllBoundingBoxes)?